Bindings/Packages/com.virtualmaker.bindings/Runtime/Bindings.PlayerPrefs.cs

```diff
@@ -0,0 +1,61 @@
+using UnityEngine;
+
+namespace VirtualMaker.Bindings
+{
+    public partial class Bindings
+    {
+        /// <summary> Binds player pref to property. </summary>
+        /// <param name="key"> Key used by <c>PlayerPrefs</c> to get and set the value. </param>
+        /// <param name="prop"> <c>Property</c> that gets the stored value then sets it when changed. </param>
+        /// <param name="defaultValue"> Value to set <c>prop.Value</c> to if a value isn't found. </param>
+        public void BindPlayerPref(string key, Property<float> prop, float defaultValue)
+        {
+            prop.Value = PlayerPrefs.GetFloat(key, defaultValue);
+            BindPlayerPref(key, prop);
+        }
+
+        /// <summary> Binds player pref to be set by an <c>IProperty</c>. </summary>
+        /// <param name="key"> Key used by <c>PlayerPrefs</c> to set the value. </param>
+        /// <param name="prop"> <c>IProperty</c> that sets the value. </param>
+        public void BindPlayerPref(string key, IProperty<float> prop)
+        {
+            Bind(prop, value => PlayerPrefs.SetFloat(key, value));
+        }
+
+        /// <summary> Binds player pref to property. </summary>
+        /// <param name="key"> Key used by <c>PlayerPrefs</c> to get and set the value. </param>
+        /// <param name="prop"> <c>Property</c> that gets the stored value then sets it when changed. </param>
+        /// <param name="defaultValue"> Value to set <c>prop.Value</c> to if a value isn't found. </param>
+        public void BindPlayerPref(string key, Property<int> prop, int defaultValue)
```

Comment:
this should be optional, falling back to "default" value if none is passed. Should be the same value as the default value from GetInt